pkg/operator/baremetal_config.go

```diff
@@ -65,3 +77,89 @@ func getBaremetalProvisioningConfig(dc dynamic.Interface, configName string) (Ba
 		ProvisioningDHCPRange:    provisioningDHCPRange,
 	}, nil
 }
+
+func getProvisioningIPCIDR(baremetalConfig BaremetalProvisioningConfig) string {
+	if baremetalConfig.ProvisioningIp != "" {
+		cidr := strings.Split(baremetalConfig.ProvisioningNetworkCIDR, "/")
+		return baremetalConfig.ProvisioningIp + "/" + cidr[1]
```

Comment:
Oh, I forgot the metal3 pod likes the IP like this. I'd at least check the len of `cidr` before reading cidr[1]. We could open ourselves up to some bad bug if cidr[1] doesn't exist.

Reply:
I'd probably also prefer parsing the CIDR rather than string manipulation, see https://play.golang.org/p/YR5SkB1wU3q

Reply:
Done.